### PR TITLE
TSLA-4256 Docs: managed cluster versions

### PR DESCRIPTION
**Pull request by gcosgrave**
Product Version(s):

Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

File: calico-cloud/get-started/connect/install-cluster.mdx [hunk 1]
```diff
@@ -26,6 +26,14 @@ Choose your install option and follow the steps.
 
 1. On the **Managed Clusters** page, click **Connect Cluster**.
 1. Add the **Cluster Name**, and select the **Cluster Type**.
+1. Select the version of {{prodname}} you want to install, we recommend the latest version, which is selected by default
```
Comment (ctauchen) [inline calico-cloud/get-started/connect/install-cluster.mdx]:
```suggestion
1. (Optional) If you must install a specific older release, select the {{prodname}} version you want to install. We always recommend the latest version, which is installed by default.
```
A suggested rewording. This strengthens the message we want to convey: Install latest always, unless you're one of the folks who are required to install a specific version for compliance.

File: calico-cloud/get-started/connect/install-cluster.mdx [hunk 2]
```diff
@@ -26,6 +26,14 @@ Choose your install option and follow the steps.
...
+   :::note
+
+   Currently the only option available to install is the latest version. More versions will become available as newer versions are released
+   and as older versions become qualified.
+
+   :::
+
```
Comment (ctauchen) [inline calico-cloud/get-started/connect/install-cluster.mdx]: I understand the idea here. But I think this note may take away more than it helps. If a user enounters a dropdown with only one version, they're unlikely to get stuck or overly frustrated. 

This time-specific note raises more questions and requires maintenance.

I'd suggest we kill the note entirely.

File: calico-cloud/get-started/upgrade-cluster.mdx [hunk 3]
```diff
@@ -7,6 +7,16 @@ description: Steps to upgrade to the latest version of Calico Cloud.
 To upgrade managed clusters to the latest version of {{prodname}}:
 
 1. In Manager UI, go to **Managed Clusters**.
-1. Highlight the cluster to upgrade in the list.
-1. Click the **Reinstall** icon.
-1. Copy the command and apply it to appropriate cluster.
+1. Click the **Actions** link on the cluster you want to reinstall, and then select  **ReInstall**.
```
Comment (ctauchen) [inline calico-cloud/get-started/upgrade-cluster.mdx]:
```suggestion
1. For the cluster you want to upgrade, select **Actions** > **Reinstall**.
```

File: calico-cloud/get-started/upgrade-cluster.mdx [hunk 4]
```diff
@@ -7,6 +7,16 @@ description: Steps to upgrade to the latest version of Calico Cloud.
...
+1. Select the version of {{prodname}} you want to install, you can select the currently installed version, or any newer supported version
+   :::note
+
+   Currently the only option available to install is the latest version. More versions will become available as newer versions are released
+   and as older versions become qualified.
+
+   :::
+
+1. Click **Next**, then copy the command and apply it to appropriate cluster.
```
Comment (ctauchen) [inline calico-cloud/get-started/upgrade-cluster.mdx]:
```suggestion
1. Click **Next**, copy the command, and apply it to appropriate cluster.
```

**Review by ctauchen — COMMENTED** [review]
Thanks for this, @gcosgrave! A few notes for you. Where appropriate, please apply globally.

File: calico-cloud/get-started/connect/install-cluster.mdx (hunk 2 above)
Comment (gcosgrave) [reply calico-cloud/get-started/connect/install-cluster.mdx]: @ozdanborne WDYT?

File: calico-cloud/get-started/connect/install-cluster.mdx (hunk 1 above)
Comment (gcosgrave) [reply calico-cloud/get-started/connect/install-cluster.mdx]: done

File: calico-cloud/get-started/upgrade-cluster.mdx (hunk 3 above)
Comment (gcosgrave) [reply calico-cloud/get-started/upgrade-cluster.mdx]: done

File: calico-cloud/get-started/upgrade-cluster.mdx (hunk 4 above)
Comment (gcosgrave) [reply calico-cloud/get-started/upgrade-cluster.mdx]: done

File: calico-cloud/get-started/connect/install-cluster.mdx (hunk 2 above)
Comment (ctauchen) [reply calico-cloud/get-started/connect/install-cluster.mdx]: FYI @malavikabala

Comment (gcosgrave) [reply calico-cloud/get-started/connect/install-cluster.mdx]: @ctauchen In the absence of @ozdanborne, @tmjd has agreed that this should be removed, which is done now.

**Comment by ctauchen** [comment]
Cloned in #917. Closing here.

**Pull request closed without merging**